user/enterprise/ssl-certificate-management.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Travis CI Enterprise SSL Certificate Management
+layout: en_enterprise
+---
+
+This page contains information relating to SSL certificates and their use in Travis CI Enterprise.
+
+By default, Travis CI Enterprise verifies SSL for all traffic between its various components. SSL verification helps ensure that traffic is delivered between components securely. It is **strongly recommended** that you configure your installation with a valid certificate from a trusted authority.
+
+If you have questions about a specific section or have an issue that is not covered then please contact us at [enterprise@travis-ci.com](mailto:enterprise@travis-ci.com) for assistance.
+
+## Using Self-signed Certificates
+
+We strongly recommend that you use a valid certificate from a trusted certificate authority when configuring your installation. That said, we recognize that there are various scenarios in which using a self-signed certificate might make more sense for your organization.
+
+> In all cases of using self-signed certificates you must [disable SSL certificate validation](#disabling-ssl-verification) in your settings. This can be done after installation is complete.
+
+> If you decide at a later time to upload your own custom certificate then this can be done at any time via the [admin dashboard](#how-to-add-or-update-certificates-after-installation).
```

Comment:
I believe that this could be open to interpretation and that this very point results in a lot of support questions for us. 🤔 

I have updated this line to remove the 'your own custom' language to make it more generic and I'll DM you to discuss the rest.

Reply:
Addressed in 51e00208.